migrations/2021-12-16-003905_names_to_lowercase/up.sql

```diff
@@ -0,0 +1,37 @@
+-- Your SQL goes here
+
+
+-- Delete person name dupes, keeping the first one
+
+delete
+from person
+where id not in (
+  select min(id)
+  from person
+  group by lower(actor_id)
+);
```

Comment:
I had to learn the hard way, that doing an in query can take days sometimes, especially on large tables. 

Use this one for dupes: https://github.com/LemmyNet/lemmy/blob/main/migrations/2021-11-22-135324_add_activity_ap_id_index/up.sql#L9